### ISSUE TITLE
available disk space in nextcloud is way less than free space in the pool

### ISSUE THREAD

**Issue by jsmith8954** [danb35/freenas-iocage-nextcloud#109]
I'm not convinced this is an issue with the script, but I wanted to reach out and see if anyone ran into the same issue (or if it's even an issue). I've searched both the Nextcloud and FreeNas forums, and can't seem to find anything similar.

The script worked great. I want to address the below issue before I start syncing a couple hundred GB's of files.  

The amount of available space on the `root` disk is nowhere close to the total free space in the Pool. As you can see in the screen below, the disk `size` is only 210 GB while the total Free space is 444.7 GB. (This matches the FreeNas Pool)

![image](https://user-images.githubusercontent.com/16658156/80168317-62101b00-8597-11ea-889e-e70771d56eb0.png)

The machine is a fresh install of FreeNAS-11.3-U2.
Nextcloud version: 18.0.3

Config file:
```
JAIL_IP="192.168.1.201"
DEFAULT_GW_IP="192.168.1.1"
POOL_PATH="/mnt/storage/nextcloud"
TIME_ZONE="America/Los_Angeles"
HOST_NAME=“nextcloud.mydomain.com"
DB_PATH=/mnt/storage/nextcloud/db
FILES_PATH=/mnt/storage/nextcloud/files
CONFIG_PATH=/mnt/storage/nextcloud/config
CERT_EMAIL=“email@gmail.com"
DNS_CERT=1
DNS_PLUGIN="cloudflare"
DNS_ENV="CLOUDFLARE_EMAIL=email@gmail.com CLOUDFLARE_API_KEY=00000000000000”
```

Before executing the script, I created a new storage pool using a new hard drive and created the following data sets on it to use in the config file.
```
storage
storage/nextcloud
storage/nextcloud/files
storage/nextcloud/config
storage/nextcloud/db
```

There's no other datasets on this pool.
![image](https://user-images.githubusercontent.com/16658156/80169023-2f672200-8599-11ea-9ac9-9e17aca73c86.png)

I skimmed the script and nothing stood out to me that would cause this.

Any thoughts, potential causes, or things to check would be appreciated. 

Thanks in advance.

**Comment by Ornias1993** [comment]
This doesn't seem script related.
There are however some issues with freespace messages being way off with ZFS in general.

Did you try actually filling it?

**Comment by danb35** [comment]
> The amount of available space on the root disk is nowhere close to the total free space in the Pool.

What is `root`, and where are you seeing the image you posted?

**Comment by Ornias1993** [comment]
Also what @danb35 is saying... I have trouble figuring out if this is even closely related to this script...

**Comment by jsmith8954** [comment]
Jumped back into is this over weekend, and did a little more discovery.
I'm confident it's a non issue with the script and will close it.

@danb35 , the image is from NextCloud itself. If logged in as an admin, go the Settings and under Administration click Overview.

The `root` drive is actually a separate pool in FreeNas where my Jails are installed.  I was initially thrown off by the image, because this drive is only 120Gb total, but NextCloud is reporting 207Gb available. 

Upon further inspection, I found that that drive has 103.5Gb available (x 2 = 207Gb).

I logged into the jail directly to check the usage of the root disk, and it is reporting correctly.

This looks like a miscalculation of the ZFS data on nextcloud.

Either way, I'm confident it's not an issue with the script, so I'll close this issue.
